proknow/Patients/Entities/Plans.py

```diff
@@ -106,12 +94,24 @@ def get_delivery_information(self):
                 plan = entities[0].get()
                 info = plan.get_delivery_information()
         """
-        self._wait_delivery_information()
         headers = {
-            'ProKnow-Key': self.data["key"]
+            "Accept-Version": "1",
+            "Authorization": 'Bearer ' + self._data["data"]["dicom_token"]
         }
-        _, content = self._requestor.get('/plans/' + self._id + '/delivery/' + self._data["data"]["delivery_tag"], headers=headers)
-        return content
+        start = datetime.datetime.now()
+        DELAY = 0.2
+        while (datetime.datetime.now() - start).total_seconds() < self._proknow.ENTITY_WAIT_TIMEOUT:
```

Comment:
The request to process a given entity. For example, you make a request like this to request that a plan be processed:

```
POST /rtv/source_name/plan
{
  "data": ["<id-that-references-plan-dicom"]
}
```

Then you have to make a request to get the delivery information:

```
GET /plan/:pid/details/:did
200
{ delivery JSON object }
```

Reply:
The POST won't trigger re-processing of an already processed DICOM plan will it?

Reply:
No, unless the patient has never been opened before and the RT Visualizer has never processed the data before (or the cached data has been cleared), it will have already been processed and cached by the RT Visualizer. These requests are idempotent, so it's safe to make them over and over again.